core/src/test/scala/unit/kafka/utils/UtilsTest.scala

```diff
@@ -151,4 +154,25 @@ class UtilsTest extends JUnitSuite {
     assertFalse("Should be unlocked", lock.isLocked)
   }
 
+  @Test
+  def testUrlSafeBase64EncodeUUID() {
+    // Test a UUID that has no + or / characters in base64 encoding
+    // a149b4a3-06e1-4b49-a8cb-8a9c4a59fa46 ->(base64)-> oUm0owbhS0moy4qcSln6Rg==
+    val clusterId1 = CoreUtils.urlSafeBase64EncodeUUID(UUID.fromString("a149b4a3-06e1-4b49-a8cb-8a9c4a59fa46"))
+    assertEquals(clusterId1.length, 22)
+    assertTrue(clusterIdPattern.matcher(clusterId1).matches())
+
+    // Test a UUID that has + or / characters in base64 encoding
+    // d418ec02-277e-4853-81e6-afe30259daec ->(base64)-> 1BjsAid+SFOB5q/jAlna7A==
```

Comment:
Seems like we can assert the expected value too instead of having it in a comment?